intellij-plugin/src/main/resources/messages/ApolloBundle.properties

```
@@ -131,4 +135,12 @@ inspection.endpointNotConfigured.quickFix=Add introspection block
 
 inspection.suppress.field=Suppress for field
 
-notification.group.apollo=Apollo
+notification.group.apollo.main=Apollo
+notification.group.apollo.telemetry=Apollo (telemetry)
+
+telemetry.optOutDialog.title=Send usage statistics to Apollo
+telemetry.optOutDialog.content=To help the team improve Apollo Kotlin, the plugin sends anonymous data to Apollo, such as which Apollo Kotlin features, options and components are used, versions of Android and Kotlin, number of modules in the project, etc.<br>\
+  No personal data or any sensitive information, such as source code or file names, are included.<br>\
+  You can opt-out in the plugin's settings at any time.
```

Agreed. I would reuse the  same wording everywhere
```suggestion
telemetry.optOutDialog.content=To improve Apollo Kotlin, the plugin sends anonymous data to Apollo, such as Gradle plugin options, version of Android and Kotlin, number of modules in the project and other technical information.<br>\
  No personal data or sensitive information, such as source code or file names are included.<br>\
  You can opt-out in the plugin's settings at any time.
```